arbeitszeit_flask/database/repositories.py

```diff
@@ -731,6 +731,15 @@ def add_worker(self, member: UUID) -> int:
             company.workers.append(member)
         return companies_changed
 
+    def remove_worker(self, member: UUID) -> int:
+        companies_changed = 0
+        member = models.Member.query.filter(models.Member.id == str(member)).first()
+        assert member
+        for company in self.query:
+            companies_changed += 1
+            company.workers.remove(member)
+        return companies_changed
```

Comment:
I know it is done the same way in add_worker but I still wonder if there is and/or will be in the future, a use case where self.query can contain more than one company? Isn't this action always triggered from exactly one company?